public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx

```diff
@@ -141,6 +142,9 @@ const getStyles = (theme: GrafanaTheme2, hasSplit: boolean) => {
   `;
 
   return {
+    splitPane: css`
+      overflow: visible !important;
```

Comment:
in panel edit, this allows the tooltip to exit the splitpane with the viz and render on top of the options sidebar. it doesnt have the desired effect in Explore, because that has layers of `overflow` rules from the `react-custom-scrollbars`. this will be solved differently in another PR.